data/locale/jack_mixer-fr.po

```diff
@@ -0,0 +1,677 @@
+# #-#-#-#-#  jack_mixer-fr.po (jack_mixer 15.1)  #-#-#-#-#
+# jack_mixer i18n message catalog template
+#
+# This file is distributed under the same license as the jack_mixer package.
+#
+# Copyright (C) 2021 Olivier (trebmuh/olinuxx) Humbert <trebmuh@tuxfamily.org>
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: jack_mixer 15.1\n"
+"Report-Msgid-Bugs-To: https://github.com/jack-mixer/jack_mixer/issues\n"
+"POT-Creation-Date: 2021-03-20 14:48+0100\n"
+"PO-Revision-Date: 2021-03-21 18:17+0100\n"
+"Last-Translator: 2021 Olivier (trebmuh/olinuxx) Humbert <trebmuh@tuxfamily.org>\n"
+"Language-Team: French\n"
+"Language: fr\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+
+#: jack_mixer/app.py:48 jack_mixer/app.py:47 jack_mixer/app.py:49
+msgid ""
+"jack_mixer is free software; you can redistribute it and/or modify it\n"
+"under the terms of the GNU General Public License as published by the\n"
+"Free Software Foundation; either version 2 of the License, or (at your\n"
+"option) any later version.\n"
+"\n"
+"jack_mixer is distributed in the hope that it will be useful, but\n"
+"WITHOUT ANY WARRANTY; without even the implied warranty of\n"
+"MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU\n"
+"General Public License for more details.\n"
+"\n"
+"You should have received a copy of the GNU General Public License along\n"
+"with jack_mixer; if not, write to the Free Software Foundation, Inc., 51\n"
+"Franklin Street, Fifth Floor, Boston, MA 02110-130159 USA\n"
+msgstr ""
+
+#: jack_mixer/app.py:118 jack_mixer/app.py:121 jack_mixer/app.py:122
+msgid "Failed to create Mixer instance."
+msgstr "Impossible de créer une instance du mixeur."
+
+# Don't translate this unless you want the monitor JACK output port name to be localized
+#: jack_mixer/app.py:123 jack_mixer/app.py:126 jack_mixer/app.py:127
+msgid "Monitor"
+msgstr "Moniteur"
```

Comment:
Are you sure it is a good idea to translate the name of the "Monitor" outputs?

This would mean that you have different port names depending on which language your environment has

I think translated port labels should be done with port meta data in the future. jack_mixer does not support setting port (or client) meta data yet, but this is one of the next things I plan to work on.

Reply:
Just to be 100% sure, we are talking about the translation of this, isn't it?

![jackmixer_monitor](https://user-images.githubusercontent.com/8705846/112050219-e1fc8780-8b50-11eb-809e-79ad4ff91b88.png)

If so, it's a good idea from my point of view to get them translated.

Reply:
Yes, I as talking about the port labels. I'm not convinced it is desirable to translate them. Be aware that this would make port label change if you load the same project on different computers with different language settings. This could cause automatic connection setup (e.g. via NSM or the qjackctl patchbay or similar tools)  to fail, because these are usually based on matching port names.

Reply:
So, you (as the dev) might want to **_not_** mark them as translatable in the first place since, if that's available for translation, translators will translate.
I'll wait for you to remove it as "translatable", then will remove this from the fr.po file.

Reply:
Port labels are now not marked as translatable anymore